todo_app/templates/index.html

```diff
@@ -33,15 +37,43 @@ <h2>Items Doing</h2>
         {% for item in items_doing %}
           <li>
             ID:{{ item.id }} - Name:{{ item.name }} - Status:Doing 
-           </li>         
+           </li>    
+           <form method="GET" action="/items/{{ item.id }}/doing">
+            <ul>
+            </ul>
+            <input type="submit" value="Move to Doing">   
+          </form>     
+          <form method="GET" action="/items/{{ item.id }}/complete">            <ul>
+            </ul>
+            <input type="submit" value="Move to Done">   
+          </form> 
+          <form method="GET" action="/items/{{ item.id }}/todo">
+            <ul>
+            </ul>
+            <input type="submit" value="Move to To Do">   
+          </form>     
         {% endfor %}
       </ul>
       <h2>Items Done</h2>
       <ul class="list-group mb-4">
         {% for item in items_done %}
           <li>
             ID:{{ item.id }} - Name:{{ item.name }} - Status:Done
-           </li>         
+           </li>        
+           <form method="GET" action="/items/{{ item.id }}/doing">
+            <ul>
+            </ul>
+            <input type="submit" value="Move to Doing">   
+          </form>     
+          <form method="GET" action="/items/{{ item.id }}/complete">            <ul>
+            </ul>
+            <input type="submit" value="Move to Done">   
+          </form> 
+          <form method="GET" action="/items/{{ item.id }}/todo">
+            <ul>
+            </ul>
+            <input type="submit" value="Move to To Do">   
+          </form>
         {% endfor %}
       </ul>
       <form method="POST" action="/add">
```

Comment:
It doesn't look like you have an `/add` endpoint, but you do have an `/items/new` endpoint. Did you mean to use that here?